tools/shoestring/shoestring/wizard/setup_file_generator.py

```diff
@@ -129,3 +136,100 @@ def patch_shoestring_config(shoestring_filepath, package_config_filepath):
 	new_config_manager = ConfigurationManager(package_config_filepath.parent)
 	new_config_manager.patch(package_config_filepath.name, config_patches)
 	shutil.copy(package_config_filepath, shoestring_filepath)
+
+
+async def patch_bootstrap_shoestring_config(bootstrap_target_path, config_filepath, include_node_key=False):
```

Comment:
unsure about the name "patch" vs something like import or apply ?

Reply:
hmm..maybe apply?